actions/handle_caches/action.yaml

```diff
@@ -0,0 +1,28 @@
+name: Test Documentation
+inputs:
+  dependencies:
+    required: true
+    type: string
+outputs:
+  cache-libint:
+    description: "Cache libint status"
+    value: ${{ steps.cacheLibint.outputs.cache-hit }}
+runs:
+  using: "composite"
+  steps:
+  - name: Obtain Source
+    uses: actions/checkout@v2
+  - if: ${{contains(inputs.dependencies, 'libint')}}
+    name: cache-libint   
+    id: cacheLibint
+    uses: actions/cache@v2
+    with:
+      path: |
+        libint-2.6.0/**/
+        install/**/
+      key: libint-gcc${{ matrix.gcc_version }}
+  - name: Test Check Cache Libint
+    env:
+      CACHE_LIBINT: ${{steps.cacheLibint.outputs.cache-hit}}
+    run: echo "helloworld 22 " $CACHE_LIBINT
```

Comment:
Please clean-up your debugging statements.

Reply:
> The names of the workflows and the names of the steps look like they have been copy/pasted. Please modify them so that they actually reflect what is going on. Also I noticed that these workflows make explicit mention of GCC. Are you sure this is going to work with our workflows (which also use clang)?

Yes. I tested it with the clang build also for Integrals (currently, except Tensorwrapper other projects in NWChemEx have 2 separated build processes using GCC and clang in c-cpp.yaml), and it worked. However, in the get_dependencies.sh that ran before the build, I used the default installation of libint that they used g++ for making the build the libint.

Reply:
> Please clean-up your debugging statements.

thank you. I cleaned the statements.